### DISCUSSION TITLE
Undefined symbols for architecture arm64 linking libmoose-opt.la

### DISCUSSION THREAD

**Discussion by Jacobfaib** [idaholab/moose#20264] [Q&A Getting Started] [answered]
See https://github.com/idaholab/moose/issues/20276

**Comment by cticenhour** [comment, marked as answer]
FYI to those who come across this thread later - Discussions can be converted to issues (and vice versa), without needing to redo or copy anything over. @Jacobfaib I will put a response in #20276
